script/Ciel/Day 5/05_01_CIEL05_4.txt

```diff
@@ -43,7 +43,7 @@ Thank goodness, she's still alive...!
 [sha:851e5dabff809d236113e08efb422ff2a2180960]{
 -- Page 679, Offset 12660.
 -- 「聞こえますか!?　苦しいだろうけど起きて、シエル先輩！」
-"Can you hear me?! I know it must hurt, but you have to get up, Ciel-senpai!"
+"Can you hear me?! I know it must have hurt, but you have to get up, Ciel-senpai!"
```

Comment:
It's more like "I know you are in pain" than "I know that what happened to you hurt", so I'd revert